.github/workflows/bench.yml

```diff
@@ -0,0 +1,40 @@
+name: Benchmarks
+
+on:
+  push:
+    branches: [ master ]
+
+jobs:
+
+  Benchmark:
+
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.7
```

Comment:
Why 3.8?

Reply:
I picked the same python version that is used for the ubuntu tests https://github.com/pyccel/pyccel/blob/master/.github/workflows/Github_pytest.yml#L14-L17 
But maybe it would make sense to use the most advanced (and therefore potentially fastest) version of python. So 3.10 ?